package.json

```diff
@@ -1,8 +1,27 @@
 {
   "private": true,
   "name": "msw-storybook-addon",
-  "workspaces": ["packages/*"],
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
     "start": "yarn workspace docs sb:start"
+  },
+  "keywords": [
+    "storybook-addon",
+    "msw",
+    "mocking",
+    "api",
+    "graphql",
+    "network",
+    "data-state"
```

Comment:
"data-state" looks quite specific. Maybe let's consider "request", "response" and "mock"? Those make much better keywords for discovery.

Reply:
I think that "data-state" is some storybook keyword since theirs a dedicated icon showing up for it on their site.

<img width="318" alt="image" src="https://user-images.githubusercontent.com/15871340/112722716-21e8b380-8f31-11eb-906e-d11656e42ab6.png">

Reply:
@itaditya got it! Didn't know that. Let's leave it then :)